crates/daphne/src/vdaf/prio3_draft09.rs

```diff
@@ -51,6 +51,28 @@ type Prio3Draft09Prepared<T, const SEED_SIZE: usize> = (
     Prio3PrepareShare<<T as Type>::Field, SEED_SIZE>,
 );
 
+pub(crate) fn shard_then_encode_draft09<V: Vdaf + Client<16>>(
```

Comment:
having `draft09` in this function name is a bit redundant, the module already has draft09 in it's name

Reply:
This is also true for the other functions in this module

Reply:
Done. I also renamed the module to `draft09` since it's also used by Pine.